test/xspec-variable.xspec

```diff
@@ -5,29 +5,212 @@
 <!--  Tags:                                                                -->
 <!--    Copyright (c) 2010 Jeni Tennison (see end of file.)                -->
 <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
-<t:description xmlns:t="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:my="http://example.org/ns/my" query="http://example.org/ns/my" query-at="xspec-tested.xql" stylesheet="xspec-tested.xsl">
+
+<!-- TODO: Make the following changes after other code gets merged onto master branch.
```

Comment:
@AirQuick: I see some relevant bug fixes on your branch. I wanted to make the connection between fixes that will enable me to make more progress here and the pull requests that I'm waiting to get merged in. But I'm pretty new to Git and GitHub, so I wasn't sure how to do that. If you have suggestions, I'll be grateful. If you prefer to spend your limited code review time on more important kinds of suggestions, I completely understand.